content/SCALE/GettingStarted/VMandAppConfigSCALE.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Configuring Virtualization and Apps in TrueNAS SCALE"
```

Comment:
The description: "" parameter is missing

Reply:
It has been added.